troubleshooting/missing-image-asset-causes-ios-publish-to-fail.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Error Publishing App to the iOS AppStore
+description: Publishing Application to the AppStore is Failing due to Missing Image Assets
+type: troubleshooting
+page_title: AppStore Publish Fails due to Missing Image Assets
+slug: invalid-image-asset-fail-ios-publish
+position:
+tags: appbuilder, publish
+teampulseid:
+ticketid: 1044213, 1044122
+pitsid:
+
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code (AppBuilder)</td>	
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Publish - AppStore</td>	
+  </tr>
+  <tr>
+    <td>Mobile OS</td>
+    <td>iOS</td>	
+  </tr>
+</table>
+
+## Description
+Publishing an app to the iOS AppStore results in an error stating `ERROR ITMS-90032: Invalid Image Path`.
+
+## Error Message
+`ERROR ITMS-90032: Invalid Image Path - No image found at the path referenced under key 'CFBundleIcons': 'icon-83.5@2x.png'`
+
+## Cause
+The AppBuilder 3.3 release included an update to the iOS Info.plist file, which added a new icon for iPad Pro. Every project created prior to this release is now referencing a missing iOS icon asset.
+
+## Solution
+Upload the iPad Pro icon (167x167 slot) that is missing or use a [single image source](http://docs.telerik.com/platform/appbuilder/cordova/configuring-your-app/assets/generate-icons-splashscreens) to generate all icons anew.
```

Comment:
I would probably state the solution like this:

Provide the missing iPad Pro icon.
* Manually upload the missing icon (the 167x167 slot). -> link to the article about manual upload
* Re-generate all icons at once for a single image source. -> link to the article about manual generation